openghg/store/_footprints.py

```diff
@@ -283,7 +288,23 @@ def read_file(
             )
             time_resolved = high_time_resolution
 
-        site = clean_string(site)
+        if satellite is None and site is None:
+            raise ValueError("Please pass either site or satellite value")
+
+        if satellite is not None:
+            satellite = clean_string(satellite)
+        else:
+            site = clean_string(site)
+
+        if domain is not None and fp_region is not None:
```

Comment:
For footprint files we wouldn't want to apply the same logic for the keywords that we needed to for the obscolumn data. For footprint `domain` is an essential keyword as this describes the area for the 2D map. For the obscolumn data we were essentially allowing `domain` to be another way to specify `obs_region` which is why we applied this logic.

Instead though we may want to include checks to ensure either `site` or `satellite` AND `obs_region` are specified so:
 - If `site` is specified, use this keyword
 - If `site` is not specified, check if `satellite` and `obs_region` are specified, otherwise raise an Error